public/js/p3/widget/viewer/SubSystemMap.js

```diff
@@ -26,17 +26,19 @@ define([
 			}
 
 			var display_reference_genomes = this.getStateParams(state);
-
 			var that = this;
-			var query = "?ne(genome_id," + state.genome_ids_without_reference + ")&eq(taxon_lineage_ids,2)&eq('reference_genome','Reference'),&select(genome_id,genome_name,reference_genome)&limit(25000)&sort(+kingdom,+phylum,+class,+order,+family,+genus)";
-			return when(request.get(PathJoin(window.App.dataAPI, "genome", query), {
+			var query = "and(in(genome_id,(" + state.genome_ids_without_reference + "))),eq(taxon_lineage_ids,2),eq(reference_genome,Reference)&select(genome_id,genome_name,reference_genome)&limit(25000)&sort(+kingdom,+phylum,+class,+order,+family,+genus)";
```

Comment:
1. We intended to remove those reference genomes from your context. i.e. if you're seeing from reference genome, you may not want to get the same genome id from this query. Just to double check.

2. you don't need ```and()``` if you have only one condition in it.